packages/typespec-client-generator-core/design-docs/flags.md

```diff
@@ -0,0 +1,35 @@
+# TCGC `tspconfig.yaml` flags
+
```

Comment:
missing `api-version`? is the `flavor` in-scoped?

Reply:
Yeah, `api-version`

`flavor` probably will go away from emitter, as we should have 1 emitter for unbranded, and 1 emitter for azure?

Reply:
oh yep! I can't believe I missed `api-version`, thank you for the catch!